### ISSUE TITLE
Stalls at Task Manager 

### ISSUE THREAD

**Issue by pcwizard7** [farag2/Sophia-Script-for-Windows#348]
### Prerequisites

- [X] Verify that this is not a Windows issue;
- [X] Refer to the [**Before running**](https://github.com/farag2/Sophia-Script-for-Windows#fire-before-running-fire).
- [X] Refer to the [**How to use**](https://github.com/farag2/Sophia-Script-for-Windows#how-to-use).
- [X] If you issue concerns the Wrapper, please mention @BenchTweakGaming in issue

### Steps to reproduce

After customising and running the Script though the wrapper app, and it stalls/stuck on changing settings for Task Manager at line around 226  

The reason I think  this is happening is that i m running the RTM build of Windows 11 22H2 

thanks

**Comment by farag2** [comment]
Yes, you're right. Open Sophia.ps1 and comment out this function due to it doesn't needed for Windows 11 22H2 and doesn't work too. I'm already discovered this "bug" and will fix in the next release soon.

https://github.com/farag2/Sophia-Script-for-Windows/blob/e49f514fc5b2c9af1d82c7b2903db3cae8c3cd48/Sophia%20Script/Sophia%20Script%20for%20Windows%2011/Sophia.ps1#L464

**Comment by pcwizard7** [comment]
Got though running the script and notice Windows10Explorer is broken but Windows 10 right click contexts menu still works

**Comment by farag2** [comment]
yes, it stopped working too on 22H2. Will be removed.

**Comment by farag2** [comment]
Fixed in 6.0.15 :)
